blog/2023-10-02-what-is-a-state-machine/index.mdx

```diff
@@ -80,41 +123,97 @@ But there are also other events that can happen:
+By mapping out all states and transitions, state machines ensure you account for every possible scenario. This minimizes unexpected behaviors (such as undesired states/transitions) and system inconsistencies because it is **mathematically impossible** for a state machine to take a transition that does not exist. You can quickly identify a transition to a state that should not go to that state. This is especially useful for complex apps, where it’s easy to miss edge cases or unexpected scenarios.
 
 ### Modularity and scalability
 
-State machines allow for isolating logic. This means that one can make changes or additions to a specific segment without overhauling the entire system. Regardless of framework, programming language, or implementation, the core abstractions remain the same: states, events, and transitions. If you need to add, change, or remove a feature, the process is to add, change, or remove states and transitions.
+State machines allow for isolating logic, meaning you can make changes or additions to a specific segment without overhauling the entire system. Regardless of framework, programming language, or implementation, the core abstractions remain the same: states, events, and transitions. If you need to add, change, or remove a feature, the process is to add, change, or remove states and transitions.
 
 ### Test path generation
 
-State machines clearly define all possible states, events, and transitions. This allows testers to simulate specific events and verify if transitions to specific states occur as expected. Since state machines are directed graphs, they act like a “map” for your app logic. This enables the automatic generation of many different “paths” through the state machine. Each of these paths represents a potential use-case, even those that may initially go unnoticed, such as edge-cases. Exhaustive test path generation is also possible, which means that every possible path through the state machine can be tested.
+State machines clearly define all possible states, events, and transitions, allowing testers to simulate specific events and verify if transitions to specific states occur as expected. Since state machines are directed graphs, they act like a “map” for your app logic. This map enables the automatic generation of many different “paths” through the state machine. Each path represents a potential use case, even those that may initially go unnoticed, such as edge cases. Exhaustive test path generation is also possible so that you can test every possible path through the state machine.
+
+_Our [export to Stories feature](/docs/export-as-code#export-formats) enables you to generate the happy paths (stories) for your app._
+
+```md
```

Comment:
Can we show this as actual markdown (e.g. in a blockquote)?

Reply:
Yes! Good point 😆